### PR TITLE
Updated PR status

### PR DESCRIPTION
**Pull request by Augustine-C**
Opened PR `cz.habarta.typescript.generator.MapEntryTest` 

Opened PR
`com.alibaba.testable.core.tool.OmniAccessorTest.should_generate_member_index`

PR accepted
`edu.illinois.starts.helpers.WriterTest.testWriteGrpahWithMultipleEdges`

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -2052,3 +2052,5 @@ https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,ript
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-spring-boot-starter,org.zalando.riptide.spring.ObjectMapperOverrideTest.shouldOverride,ID,,,
 https://github.com/zalando/riptide,8277e11fc069d8e24df0d233ef2577cc75659b75,riptide-spring-boot-starter,org.zalando.riptide.spring.url.UrlResolutionTest.shouldAppendUrl,UD,,,
 https://github.com/zhangxd1989/spring-boot-cloud,e3966d7cefa4fa429d13bbc8de7f4dafbae0de35,registry,cn.zhangxd.registry.ApplicationTests.catalogLoads,ID,,,
+https://github.com/TestingResearchIllinois/starts,76323ed1eb39bb7b4aea9797c9f081f3dd521ebd,.,edu.illinois.starts.helpers.WriterTest.testWriteGrpahWithMultipleEdges,ID,Accepted,https://github.com/TestingResearchIllinois/starts/pull/89,
```
Comment (darko-marinov) [inline pr-data.csv:2055]: You need to sort these two lines that you added at the end: https://github.com/TestingResearchIllinois/idoft#to-contribute-a-newly-detected-flaky-test

File: pr-data.csv [hunk 2]
```diff
@@ -1859,9 +1859,9 @@ https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c
...
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testDefaultShapes,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testListOfMapEntry,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testOverriddenShapes,ID,,,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testDefaultShapes,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/717,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testListOfMapEntry,ID,Opened,,
```
Comment (darko-marinov) [inline pr-data.csv:1863]: Missing PR link for two lines here.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
The contributions look promising, but the IDoFT checker failed, so follow up on the failure messages.

File: pr-data.csv (hunk 1 above)
Comment (Augustine-C) [reply pr-data.csv]: Sorry that I missed the guidance for adding new tests. I have fixed that now.

File: pr-data.csv (hunk 2 above)
Comment (Augustine-C) [reply pr-data.csv]: I have added the PR link.

Comment (darko-marinov) [reply pr-data.csv]: Sorry I didn't realize that PR is only for the first test and not the other two. Please remove `Opened` from these two for now. Hopefully the 717 PR will be accepted, so you can open another for the other two tests.

Comment (Augustine-C) [reply pr-data.csv]: I just removed the opened label and moved the link to the note column.

**Pull request merged**